modules/proxy.js

```diff
@@ -25,16 +52,39 @@ module.exports = async ({msg,bot,members,cfg}) => {
 		if(replace.length < 2) replace = [];
 	
 		if(!replace[0]) {
+			let found = false;
 			for(let t of members) {
 				let res = bot.checkMember(msg, t, clean);
 				if(res >= 0) {
-					replace.push([msg, cfg, t, t.show_brackets ? msg.content : msg.content.substring(t.brackets[res*2].length, msg.content.length-t.brackets[res*2+1].length)]);
+					found = true;
+					let modified = msg.content.replace(matchLineEndsWithAuto, "");
+					modified =t.show_brackets ? modified : modified.substring(t.brackets[res*2].length, modified.length-t.brackets[res*2+1].length);
+					//if message ends in -tul!auto, set the auto proxy flag
+					if(msg.content.match(matchPhraseEndsWithAuto)){
+						//replace all spaces before -tul!auto and -tul!auto itself in the message
+						setAutoProxyMember = t;
+						setAutoProxy = true;
+					}
+					replace.push([msg, cfg, t, modified]);
 					break;
 				}
 			}
+			if(!found && automember !== undefined && clean.length > 0){
+				usingautomember = true;
+				let t = automember;
+				let modified = msg.content.replace(matchLineEndsWithAuto, "");
+				//if message ends in -tul!auto, set the auto proxy flag
+				if (msg.content.match(matchPhraseEndsWithAuto)) {
+					//replace all spaces before -tul!auto and -tul!auto itself in the message
+					setAutoProxyMember = null;
+					setAutoProxy = true;
```

Comment:
It doesn't seem like this should be set to true here

Reply:
This is for the case where a person types a non-poxy message and appends -tul!auto to it.  It will clear the current auto proxy when the member is null and "setAutoProxy" is true.